pom.xml

```diff
@@ -46,8 +46,8 @@
     </issueManagement>
 
     <properties>
-        <jenkins.version>1.580</jenkins.version>
-        <jenkins-test-harness.version>1.580</jenkins-test-harness.version>
+        <jenkins.version>1.615</jenkins.version>
```

Comment:
I would vouch for using LTS baseline like `1.625` since you go beyond the Java 6 compat threshold anyway